packages/algob/src/types.ts

```diff
@@ -450,14 +466,19 @@ export interface AlgobDeployer {
     scParams: Object, // Parameters
     signer: Account
   ) => Promise<LsigInfo>
+  deploySSC: (
+    approvalProgram: string,
+    clearProgram: string,
+    flags: SSCDeploymentFlags,
+    payFlags: TxParams) => Promise<SSCInfo>
   /**
-     Returns true if ASA or ACS were deployed in any script.
+     Returns true if ASA or ASC were deployed in any script.
      Checks even for checkpoints out of from the execution
      session which are not obtainable using get methods.
```

Comment:
can you check if grammer is correct for lines 476, 477 as well? Thanks.

Reply:
hmm, can you please suggest something ?

Reply:
`Checks even for checkpoints from which the execution
     session are not obtainable using get methods.`